openfe/tests/protocols/openmm_md/test_plain_md_protocol.py

```diff
@@ -188,28 +189,43 @@ def test_dry_run_gaff_vacuum(benzene_vacuum_system, vac_settings, tmpdir):
         stateB=benzene_vacuum_system,
         mapping=None,
     )
-    unit = list(dag.protocol_units)[0]
```

Comment:
This is why we should slowly rename things to offunit and ommunit, but maybe not in this PR.